lighthouse-core/computed/unused-javascript-summary.js

```diff
@@ -127,9 +127,9 @@ class UnusedJavascriptSummary {
       let offset = lineOffsets[mapping.lineNumber];
 
       offset += mapping.columnNumber;
-      const lastColumnOfMapping =
-        // @ts-expect-error: We will upstream lastColumnNumber to CDT eventually.
-        (mapping.lastColumnNumber - 1) || lineLengths[mapping.lineNumber];
```

Comment:
`mapping.lastColumnNumber` is sometimes be undefined (for the last mapping of each line), and in that case this value falls back to the line length (acknowledgement: this is mixing data define in the _map_ and data from the _script content_).

arguably this should be calculated in `computeLastGeneratedColumns`, but 1) `TextSourceMap` doesn't know about the script content and 2) each mapping doesn't know how long of a range it covers, so there is no way to know what to set the lastColumnNumber of the last mapping on a line.